### ISSUE TITLE
Failed to generate behavior

### ISSUE THREAD

**Issue by wilkub** [ShikyoKira/Project-New-Reign---Nemesis-Main#338]
Got this error, tried a few different sse versions, then purged mods in vortex and did a completely clean re-install with absolutely nothing other than skyrim and nemesis and got the exact same error.
Am I missing something?
Is a problem that skyrim is in c://program files maybe?

```
Checking engine version
Engine is up to date
Version Code: 2445-570

Initializing behavior generation

Mod Checked 1: nemesis

ERROR(5001): Invalid format has been detected in "animationsetdatasinglefile.txt". Please contact the template creator or mod author
Project: DefaultFemaleData\DefaultFemale.txt
Header: DragonMountedMRhSolo_ShieldOrTorch.txt

Failed to generate behavior
```

**Comment by ShikyoKira** [comment]
Try install this manually not through vortex and then run it

**Comment by wilkub** [comment]
yes, I did that, this happens even with completely clean install of skyrim SE with nemesis unpacked into it.

**Comment by wilkub** [comment]
Moved to skyrim se to a different drive, still the same.

**Comment by ShikyoKira** [comment]
post your patchlog.txt and the `animationsetdatasinglefile.txt` located in `nemesis_engine/temp_behaviors`

**Comment by wilkub** [comment]
[PatchLog.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/5005501/PatchLog.txt)
[animationsetdatasinglefile.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/5005503/animationsetdatasinglefile.txt)

**Comment by ShikyoKira** [comment]
why aren't you using the latest version? Please update. I don't support any versions older than the latest version

**Comment by wilkub** [comment]
v0.84-beta is the newest, right?
here are the logs from that, exact same thing
[PatchLog.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/5009768/PatchLog.txt)
[animationsetdatasinglefile.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/5009769/animationsetdatasinglefile.txt)

**Comment by ShikyoKira** [comment]
good, at least we are on to something. Thanks for the report. I'll look into it

**Comment by ShikyoKira** [comment]
I do noticed there are some issues with this animationsetdatasinglefile.txt of yours. Do you mind sending me another file, `Nemeiss_animationsetdatasinglefile.txt` located in `meshes` folder. After this, I think it should be sufficient for me to begin the investigation and diagnosis

**Comment by wilkub** [comment]
[nemesis_animationsetdatasinglefile.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/5011359/nemesis_animationsetdatasinglefile.txt)

**Comment by ShikyoKira** [comment]
Thank you. I'll begin investigating it as soon as i have the time to deal with this

**Comment by AMGarkin** [comment]
I have the same issue with file generated by "update engine" button (file dumped to tmp_behaviors is exactly the same as wilkub posted above). Tested with clean installation of Skyrim SE + Nemesis v0.84 beta on Czech version of Win10 (according to log original poster have also Czech version of Windows). 

File contains has exactly the same data as the working one from issue #155 + reference to NemesisData\NemesisProject.txt. The problem is that data are in different order.
The problem is that different languages sometimes have exceptions in sorting, for example in Czech words starting with "ch" are sorted after words starting with "h". If I try to run Nemesis in english environment, updating works correctly.

Workaround:
Download [Locale Emulator](https://github.com/xupefei/Locale-Emulator/releases/latest), unpack it anywhere on your disk and run LEInstaller.exe. Then right-click on Nemesis Unlimited Behavior Engine.exe, select Locale Emulator->Modify Application Profile. Select English language, your timezone and then click create shortcut button. It will create desktop shortcut to Nemesis. If you run Nemesis using this shortcut, it will work correctly (because locale will be set to English).

**Comment by JamesFreezeCZ** [comment]
> I have the same issue with file generated by "update engine" button (file dumped to tmp_behaviors is exactly the same as wilkub posted above). Tested with clean installation of Skyrim SE + Nemesis v0.84 beta on Czech version of Win10 (according to log original poster have also Czech version of Windows).
> 
> File contains exactly the same data as the working one from issue #155 + reference to NemesisData\NemesisProject.txt but in different order.
> The problem is that different languages sometimes have exceptions in sorting, for example in Czech words starting with "ch" are sorted after words starting with "h". If I try to run Nemesis in english environment, updating works correctly.
> 
> Workaround:
> Download [Locale Emulator](https://github.com/xupefei/Locale-Emulator/releases/latest), unpack it anywhere on your disk and run LEInstaller.exe. Then right-click on Nemesis Unlimited Behavior Engine.exe, select Locale Emulator->Modify Application Profile. Select English language, your timezone and then click create shortcut button. It will create desktop shortcut to Nemesis. If you run Nemesis using this shortcut, it will work correctly (because locale will be set to English).

Also using Win10 with Czech locale. With English locale it works.
I completely forgot that Locale Emulator emulates all locales not only Japanese.

**Comment by Skadi999** [comment]
Hello, I'm also using Win10 with Czech locale. The Locale Emulator didn't work for me, I'm not sure if I installed it incorrectly or what but Nemesis had a different problem with the Locale Emulator. Anyway, I think I found a different workaround. In win10, in the settings you should go to the Time & Language section. There you will find Region (on the left) and you need to change the 2 top settings (Country/region and Local Format). I changed the first one to USA and the second one to English (USA) then ran Nemesis and it worked fine!

**Comment by KilliFilli25** [comment]
@Skadi999 Kamaráde, bláznil jsem asi tak 4 hodiny, děkuji ti :D ( Friend, i was going mad for like 4 hours, thank you :D )
